src/platforms/common/configuration/options.mdx

```diff
@@ -372,6 +372,12 @@ _(New in version 6.0.0)_
 
 </ConfigKey>
 
+<ConfigKey name="attach-screenshot" supported={["iOS"]}>
```

Comment:
correcting format so this will display on the page for iOS
```suggestion
<ConfigKey name="attach-screenshot" supported={["apple.ios"]}>
```

Reply:
Oh, this is how you do it. Okay!!